plugins/org.jkiss.dbeaver.data.office/src/org/jkiss/dbeaver/data/office/export/DataExporterXLSX.java

```diff
@@ -225,6 +226,15 @@ public void init(IStreamDataExporterSite site) throws DBException {
         this.rowCount = 0;
         this.sheetIndex = 0;
 
+        String df = CommonUtils.toString(properties.get(PROP_DATE_FORMAT), DEFAULT_DATE_FORMAT);
+        this.dateFormatString = df;
+        if (CommonUtils.isEmpty(df)) {
+            styleDate.setDataFormat((short) 14);
```

Comment:
What's `14`? Can you clarify the meaning behind it?

Reply:
fixed